### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by kamilmateusiak**
## v0.4.0

 _Release: 2019-05-20_

 ##### New Features

 - `Card` and `Divider` components

 ##### Refactors

 - documentation styles updated
- adding typescript types and support for other html properties to `Tabs`, `TabsWrapper` and `TabsList` components
- `footer` props of `Modal` component is now optional

 ##### Deprecations

 - `Tabs` main handler `onSelect` changed to `onClick`. Right now `onSelect` property is handling HTML `onselect` event

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 All notable changes to this project will be documented in this file.
 
+## v0.4.0
+
+_Release: 2019-05-20_
+
+##### New Features
+
+- `Card` and `Divider` components
```
Comment (dprzybylek) [inline CHANGELOG.md]: divider is not listed in docs (cause it was developed ad hoc), should we annouced this in changelog?

Comment (kamilmateusiak) [reply CHANGELOG.md]: it doesn't have a dedicated page but AFAIR it's used in `Card` page. IMHO we should include it here. What ya think?

Comment (dprzybylek) [reply CHANGELOG.md]: i have mixed feelings, could stay but we should provide a dedicated page in docs in the nearest future.

Comment (kamilmateusiak) [reply CHANGELOG.md]: Ok, I will add it tomorrow :)

**Pull request merged**